zstacklib/zstacklib/storage/iscsi/session.py

```
@@ -0,0 +1,472 @@
+        '{} --name node.session.auth.username --value={}'.format(base_cmd, username)
+    )
+    if r != 0:
+        raise ChapAuthError(
+            portal=portal_str, target=iqn,
+            message="Failed to set CHAP username: {}".format(e),
+            return_code=r
+        )
+    
+    # Set password (quote it for safety)
+    quoted_password = linux.shellquote(password)
+    r, o, e = bash.bash_roe(
```

Ruff (0.15.2)

[warning] Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)